### PR TITLE
ElasticExplore API updates Dec 2020

### PR DESCRIPTION
**Pull request by alfinhaji**
**Checklist**

- [ ] tests added
- [ ] commits conform to the [Commit message format](https://github.com/qlik-oss/gopherciser/blob/master/.github/CONTRIBUTING.md#commit)
- [ ] documentation updated

**Squash commit message**

Descriptive text which shall be used as squash commit message to master.

**Info**

Optional informative text (not to be included in commit message) relevant to reviewers.

### CODE REVIEW

File: scenario/elasticexplore.go [hunk 1]
```diff
@@ -187,7 +187,7 @@ func (settings ElasticExploreSettings) Execute(sessionState *session.State, acti
 		}
 	}
 	if space != nil && space.Links.Assignments.Href != "" {
-		sessionState.Rest.GetAsync(space.Links.Assignments.Href, actionState, sessionState.LogEntry, nil)
+		sessionState.Rest.GetAsync(space.Links.Assignments.Href, actionState, sessionState.LogEntry, nil) // Need to add ?limit=100 to space.Links.Assignments.Href
```
Comment (DnlLrssn) [inline scenario/elasticexplore.go:190]: sessionState.Rest.GetAsync(space.Links.Assignments.Href, actionState, sessionState.LogEntry, nil)
How exectly does the query in href look?
There's the "easy" way if we're sure how it looks with adding 
space.Links.Assignments.Href + "?limit=100" , if we know it doesn't already have any parameters
or space.Links.Assignments.Href + "&limit=100" 
to be sure to always work we could parse it to Url, add the parameter and then back to string.

**Comment by alfinhaji** [comment]
The only outstanding issue with ElasticExplore validation is issue https://github.com/qlik-oss/gopherciser/issues/157.  Otherwise I have validated the following:

Defaults (no settings defined) ✔️   - API has been updated, new URL parameter introduced urlParams["resourceType"] = "app,qvapp,qlikview,genericlink,sharingservicetask"

Paging on/off ✔️ 
Sorting by created, modified and name ✔️  (minor update to the name API sorting, -name to %2Bname)
Space (space name) ✔️   added limit=100 parameter to assignments call
Spaceid - see [issue ](https://github.com/qlik-oss/gopherciser/issues/157)
Tag and tagids ✔️

**Review by DnlLrssn — APPROVED** [review]
lgtm, I will take a look at the issue

**Comment by DnlLrssn** [comment]
I have updated this branch with fix for space ID, don't know if you want to verify that first, if not, just merge this PR.

**Pull request merged**